### ISSUE TITLE
Parsing issue with Typescript

### ISSUE THREAD

**Issue by SPoint42** [returntocorp/ocaml-tree-sitter#132]
This code failed to be parse : 

```typescript
export default interface UserCredential {
  idToken: string;
  user: any;
}
```

I've check, in https://github.com/tree-sitter/tree-sitter-typescript/blob/master/common/define-grammar.js, and think this is  a not supported statement in the parser. But not sure as I don't know really well tree-sitter

**Comment by aryx** [comment]
@mjambon can you create the issue upstream if it comes from max?

**Comment by mjambon** [comment]
Hi @SPoint42! Sorry for the delay. There's a pending pull request that should fix this: https://github.com/tree-sitter/tree-sitter-typescript/pull/125/files

**Comment by SPoint42** [comment]
any ETA ?

**Comment by aryx** [comment]
As soon as it's merged in the tree-sitter repo. Feel free to ping the PR martin linked above.

**Comment by nbrahms** [comment]
@mjambon is this done?

**Comment by mjambon** [comment]
yes, this is fixed in production (semgrep 0.56 or above)!
